packages/node-dev/tsconfig.json

```diff
@@ -3,10 +3,35 @@
 	"compilerOptions": {
 		"outDir": "dist",
 		"types": ["node"],
+		"lib": ["es2019", "es2020", "es2022.error"],
+		"strict": true,
+		"module": "commonjs",
+		"moduleResolution": "node",
+		"target": "es2019",
 		"preserveSymlinks": true,
+		"removeComments": true,
+		"forceConsistentCasingInFileNames": true,
+		"noImplicitAny": true,
+		"noImplicitReturns": true,
+		"strictNullChecks": true,
+		"preserveConstEnums": true,
+		"esModuleInterop": true,
+		"resolveJsonModule": true,
+		"incremental": true,
+		"declaration": true,
+		"sourceMap": true,
+		"skipLibCheck": true,
 		// TODO: remove all options below this line
```

Comment:
If this is not necessary, delete it, please.

Reply:
Deleted, thanks!.